Tribler/Test/Core/CreditMining/test_creditmining_sys.py

```diff
@@ -357,19 +356,10 @@ def test_chn_lookup(self):
         self.boosting_manager.add_source(dispersy_cid)
         chn_obj = self.boosting_manager.get_source_object(dispersy_cid)
 
-        def _load(torrent, callback=None):
-            if not isinstance(torrent, CollectedTorrent):
-                torrent_id = 0
-                if torrent.torrent_id <= 0:
-                    torrent_id = self.session.lm.torrent_db.getTorrentID(torrent.infohash)
-                if torrent_id:
-                    torrent.update_torrent_id(torrent_id)
-
-                torrent = CollectedTorrent(torrent, self.tdef)
-            if callback is not None:
-                callback(torrent)
-            else:
-                return torrent
+        def _load(_):
+            defer_ret = defer.Deferred()
+            defer_ret.callback(self.tdef)
+            return defer_ret
```

Comment:
`defer_ret` does not have any callback so it has no effect?

If you want to return a Deferred that fires immediately, please consider using `succeed` (http://twistedmatrix.com/documents/current/api/twisted.internet.defer.succeed.html)